routes/users.js

```diff
@@ -0,0 +1,85 @@
+// Routes relating to accounts here
+const express = require('express');
+
+const userRouter = express();
+const admin = require('../firebase');
+const { db } = require('../server/db');
+const { isAlphaNumeric, keysToCamel } = require('../common/utils');
+
+userRouter.use(express.json());
+
+// Get all users
+userRouter.get('/', async (req, res) => {
+  try {
+    const users = await db.query(`SELECT * FROM users`);
+    res.status(200).json(keysToCamel(users));
+  } catch (err) {
+    res.status(500).send(err.message);
+  }
+});
+
+// Get a specific user by ID
+userRouter.get('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    isAlphaNumeric(id); // ID must be alphanumeric
+
+    const user = await db.query('SELECT * FROM users WHERE id = $(id)', { id });
+    res.status(200).json(keysToCamel(user) ?? []);
+  } catch (err) {
+    res.status(400).send(err.message);
+  }
+});
+
+// Delete a specific user by ID, both in Firebase and PNP DB
+userRouter.delete('/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    isAlphaNumeric(id); // ID must be alphanumeric
+
+    // Firebase delete
+    await admin.auth().deleteUser(id);
+    // DB delete
+    await db.query('DELETE FROM users WHERE id = $(id)', { id });
+
+    res.status(200).send(`Deleted user with ID: ${id}`);
```

Comment:
In the case we need the user data, let's retrieve that data in the SQL and return it in the response body.